app/components/add-validations-to-form.js

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { GRAPHS } from '../controllers/formbuilder/edit';
+import { ForkingStore } from '@lblod/ember-submission-form-fields';
+import { getLocalFileContentAsText } from '../utils/get-local-file-content';
+import { FORM, RDF } from '../utils/rdflib';
+import { areValidationsInGraphValidated } from '../utils/validation-shape-validators';
+
+export default class AddValidationsToFormComponent extends Component {
+  @tracked builderStore;
+  @tracked builderForm;
+  @tracked formTtlCode;
+  @tracked showRdfForm = false;
```

Comment:
Nitpick: it's usually better to use derived state if possible. Either by moving the logic to an ember-concurrency task (which has a running state) or by using a getter that returns true if builderStore and builderForm aren't falsy.

Reply:
Created a `getter` for it + renamed it to `canShowRdfForm`